tox.ini

```diff
@@ -43,7 +43,20 @@ basepython = python3.9
 skip_install = true
 skipsdist = true
 commands =
+    pip-compile -U --generate-hashes requirements-bandit.in
```

Comment:
I added `bandit` hashed requirements-bandit.txt generation but I did not update line 49 for the test-requirements.txt generation due to Python 2 using test-requirements.txt.  

Bandit has older versions which will run on Python 2.7 but I do not see mentions of Python 2.6.